### ISSUE TITLE
Documentation feedback on /docs/connect/index.md

### ISSUE THREAD

**Issue by amotl** [crate/crate-clients-tools#141]
### Documentation feedback

 
- Page title: Connect to a CrateDB cluster
- Page URL: https://cratedb.com/docs/crate/clients-tools/en/latest/connect/index.html
- Source: https://github.com/crate/crate-clients-tools/blob/main/docs/connect/index.md

---

Left column is too small. Can you imagine what change may have caused this, and/or could you provide a quick fix, @msbt?

/cc @kneth

**Comment by msbt** [comment]
@amotl a real fix would be adjusting the width of the column in the markdown directly, e.g. https://github.com/crate/crate-clients-tools/blob/main/docs/connect/index.md?plain=1#L138 - we could do `:widths: 2 3 5 2` instead of `:widths: 1 3 6 2` which would give the first column more space. Other than that, we could tweak the main container to have 100% width, depending on your screen size this is currently not fully utilized. The recent updates introduced `overflow-wrap: break-word;`, which breaks the words if there's no space available.

What both solutions don't fix is a proper mobile view since the columns never collapse or get more width than the available screensize. We could give the parent element an `overflow-y: scroll;` and remove the percentage width of the columns, so they will use the size that they need and you can scroll horizontally on mobile.

**Comment by amotl** [comment]
> adjusting the width of the column in the markdown directly, - we could do `:widths: 2 3 5 2` instead of `:widths: 1 3 6 2`

Ah right, it is clear then: Column width 1 is not very wide. Apologies for the noise.

**Comment by msbt** [comment]
Yep, but that doesn't solve the responsiveness on smaller screens

**Comment by amotl** [comment]
Irgendwas ist immer. "auto" to the rescue?

**Comment by msbt** [comment]
> Irgendwas ist immer. "auto" to the rescue?

Tried auto and a bunch of other stuff today, but flexbox is not quite flexible when it comes to same size rows/columns, will revisit in one of those winter nights you suggested ;)
